packages/rich-text/src/helpers/editor.ts

```diff
@@ -67,7 +67,7 @@ export function getElementFromCurrentSelection(editor) {
 
   return Array.from(
     Editor.nodes(editor, {
-      at: editor.selection.focus,
+      at: editor.selection,
```

Comment:
Would you be so kind to share your learnings as code comments? I don't personally know why it has to be this way.